yarn.lock

```diff
@@ -3018,7 +3018,7 @@
     "@types/minimatch" "*"
     "@types/node" "*"
 
-"@types/google-protobuf@^3.10.0", "@types/google-protobuf@^3.7.2":
+"@types/google-protobuf@^3.10.0":
```

Comment:
Maybe revert this, or double-check with FE folks?

Reply:
I just opened this PR to address this issue: https://github.com/gravitational/teleport.e/pull/956

Reply:
Thanks everyone.

Reply:
Merged, updating `e` to commit a9eb334372153d554dec62f47e4d3d7b24cf1ce8 should keep us consistent with version `3.10.0`

Reply:
Many thanks!

Reply:
rebased to master which has the `e` ref ahead of a9eb334372153d554dec62f47e4d3d7b24cf1ce8, and reverted this yarn.lock change

Reply:
@nklaassen This change should still be committed, why did you revert it?

Reply:
@jentfoo I just got confused 😅 I didn't think this PR was supposed to have any yarn.lock changes. But you're right, there should be no more reference to google-protobuf@^3.7.2. I've now rebased this branch without that revert commit.

Reply:
Thank you! :raised_hands: